src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,101 @@
+import classNames from 'classnames';
+import { Person } from '../types';
+import { Loader } from './Loader';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  people: Person[],
+  errorMessage: string,
+  loading: boolean,
+  selectedPersonSlug: string | undefined,
+};
+
+export const PeopleTable: React.FC<Props> = (
+  {
+    people, errorMessage, loading, selectedPersonSlug,
+  },
+) => {
+  function parentName(name: string | null) {
+    return people.find(p => p.name === name) || null;
+  }
+
+  return (
+    <div className="block">
+      {loading
+        ? (
+          <div className="box table-container"><Loader /></div>
+        )
+        : (
+          <div className="box table-container">
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              {errorMessage}
+            </p>
```

Comment:
the p tag should be rendered conditionally , only when there is an error ( inspect it and you will find that it is always there)

Reply:
It is conditional with loading :)